tardis/io/util.py

```diff
@@ -138,6 +138,57 @@ def yaml_load_file(filename, loader=yaml.Loader):
         return yaml.load(stream, Loader=loader)
 
 
+def parse_species_list(sdec_plotter, data, species_list, packets_mode, nelements=None):
```

Comment:
why does it take a plotter argument? What is the purpose of this function?

Reply:
because it relies on some methods and attributes within sdec_plotter to parse and organize the species list.